build.gradle

```diff
@@ -5,6 +5,7 @@ plugins {
     id "com.jfrog.bintray"              version "1.8.0"     apply false
     id "com.github.kt3k.coveralls"      version "2.8.2"     apply false
     id 'com.github.ben-manes.versions'  version '0.17.0'
+    id "nebula.optional-base"           version "3.3.0"
```

Comment:
If you add a plugin here it gets applied. Our base project is not a java project and does not need the plugin.
`apply false` prevents that.

Reply:
Ah OK I'll add `apply false` to the plugin.